cli/ballerina-cli/spotbugs-exclude.xml

```diff
@@ -45,6 +45,14 @@
         <Class name="io.ballerina.cli.task.RunTestsTask"/>
         <Bug pattern="NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE"/>
     </Match>
+    <Match>
+        <Class name="io.ballerina.cli.task.RunNativeImageTestTask"/>
+        <Bug pattern="NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE"/>
+    </Match>
+    <Match>
+        <Class name="io.ballerina.cli.utils.NativeUtils"/>
+        <Bug pattern="URF_UNREAD_FIELD"/>
+    </Match>
```

Comment:
What are these exclusions for?

Reply:
The classes defined in the NativeUtils have an unread field spotbug error. This seems to happen for some of the classes in the utils section. Even though they are being accessed anyway... Not sure how to get rid of the spotbug